### PR TITLE
Added classifier to "+rt" meta

### PR DESCRIPTION
**Pull request by MikhailLipanin**
#1010

### CODE REVIEW

File: eo-maven-plugin/src/main/java/org/eolang/maven/ResolveMojo.java [hunk 1]
```diff
@@ -273,10 +287,19 @@ private static Optional<Dependency> artifact(final Path file) throws IOException
...
+            ret = String.format(
+                "%s:%s:%s",
+                dep.getGroupId(), dep.getArtifactId(), dep.getVersion()
+            );
+        } else {
+            ret = String.format(
+                "%s:%s:%s:%s",
+                dep.getGroupId(), dep.getArtifactId(), dep.getClassifier(), dep.getVersion()
+            );
+        }
```
Comment (Graur) [inline eo-maven-plugin/src/main/java/org/eolang/maven/ResolveMojo.java:301]: @MikhailLipanin Can we simplify this part? Just by calculating the `classifier` and pass it to `String.format`

**Review by Graur — CHANGES_REQUESTED** [review]
@MikhailLipanin Good one! I suggest just one minor fix

**Review by Graur — APPROVED** [review]
@yegor256 please, have a look

**Comment by yegor256** [comment]
@rultor hello

**Pull request merged**